README.md

```diff
@@ -1,9 +1,32 @@
-Meetup #1
+High To Low workshop #1
 =========
 
-Example code from first meetup.
 
+Téma workshopu: Syntax a Typy (String)
+---------
+
+Z high level jazykov (ako napríklad PHP alebo Python) sme zvyknutí na dynamický typ string, ktorý sa dá okamžite a ľahko manipilovať. Keďže jazyku Rust ide o performance, musíme zaobchádzať zo stringom na nižšej úrovni. Čo to znamená?
+
+Rust má dva typy, ktoré môžu obsahovať stringové dáta. Prvým je `&str` a `String`. V rôynzch ukážkach kódu Rustu sa môžeme stretnúť s týmto zápisom:
```

Comment:
Rust má viac ako dva typy stringov. (napr ešte `OsStr`, `OsString`, `CStr`, `CString`... ;)
Možno to označiť ako základné typy?

> Prvým je `&str` a druhým je `String`

s/rôynzch/rôznych/